api/capi/src/nnstreamer-capi-single.c

```diff
@@ -75,9 +75,9 @@ G_LOCK_DEFINE_STATIC (magic);
   g_mutex_unlock (&single_h->lock);
 
 /**
- * @brief Default time to wait for an output in appsink (3 seconds).
+ * @brief Default time to wait for an output in appsink (10 seconds).
  */
-#define SINGLE_DEFAULT_TIMEOUT 3000
+#define SINGLE_DEFAULT_TIMEOUT 10000
```

Comment:
If default value should be changed, we may need another ACR for release.
Is it possible to remain this value, and change only the testcase? (@again4you plz see this change)

Reply:
> Is it possible to remain this value, and change only the testcase?

@jaeyun-jung I think this can be done.

Reply:
Done.